whisper/stt/__init__.py

```diff
@@ -67,3 +70,19 @@ def set_num_threads(n):
     NUM_THREADS = int(NUM_THREADS)
 # For Torch, we will set it afterward, because setting that before loading the model can hang the process (see https://github.com/pytorch/pytorch/issues/58962)
 set_num_threads(1)
+
+if os.environ.get("USE_ACCURATE","true").lower() in ["true", "1"]:
+    USE_ACCURATE = True
+elif os.environ.get("USE_ACCURATE","true").lower() in ["false", "0"]:
+    USE_ACCURATE = False
```

Comment:
factorization:
```
env_use_accurate = os.environ.get("USE_ACCURATE","true")
````
and...
Shouldn't we switch default to false given the little difference we see in the WER in the benchmark?